OpenRA.Mods.Common/TraitsInterfaces.cs

```diff
@@ -420,6 +420,12 @@ public interface IMove
 		bool CanEnterTargetNow(Actor self, Target target);
 	}
 
+	public interface IWrapMove
+	{
+		Activity WrapMove(Func<Actor, Activity> moveInner);
+		bool CanWrapOrder(TargetModifiers modifiers);
```

Comment:
`CanWrapMoveOrder` for consistency please.

Reply:
Fixed.